v2/test/load/routes.test.js

```diff
@@ -5,13 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import loadRoutes from '@lib/load/routes';
 import loadSetup from '../loadSetup';
 
 describe('loadRoutes', () => {
   test('simple website', async () => {
-    const props = await loadSetup('simple');
-    const {routesPaths} = await loadRoutes(props);
```

Comment:
`loadSetup()` already calls `loadRoutes()` internally, there's no need to call it again.